website/docs/components/inputs/kafka_franz.md

```diff
@@ -62,6 +62,14 @@ input:
       root_cas_file: ""
       client_certs: []
     sasl: []
+    credentials:
+      profile: ""
+      id: ""
+      secret: ""
+      token: ""
```

Comment:
instead of explicitly defining these in the kafka yaml, a user can simply just pass in `AWS_ACCESS_KEY` and `AWS_SECRET_KEY`. And it will be caught due to the behavior of this:
https://github.com/ekeric13/benthos/blob/9595631f6c0d1514ab9bc5006e9d38b7421cabc3/internal/impl/aws/session.go#L52

correct?

Reply:
Yep, it's fine to leave the credentials fields empty or undefined.